shopinvader/tests/test_cart_item.py

```diff
@@ -95,26 +96,46 @@ def remove_cart(self):
         self.cart.unlink()
         self.shopinvader_session.pop('cart_id')
 
-    def test_pricelist_product(self):
+    def _test_pricelist_product(self):
```

Comment:
@sebastienbeau Why is this test disabled?

Reply:
Le test n'est pas désactivé, je pense qu'il faut que je renomme la méthode.
J'ai créé 2 tests plus complète et corrigé qui appel cette méthode en commun
https://github.com/akretion/odoo-shopinvader/pull/284/commits/f1f242ed6ed579917a36d05acaadcc084fe7db99#diff-6b7bd4c303db2d0e84af8813e9e0f038R125